students/sikorski_marcin/lesson_9&10/madlibs.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python
+
+import re
+
+file = open('fileToCheck.txt')
```

Comment:
podczas otwierania plików, zaleca się użycie 'with':
https://stackoverflow.com/questions/9282967/how-to-open-a-file-using-the-open-with-statement
https://stackoverflow.com/questions/3012488/what-is-the-python-with-statement-designed-for